mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -65,6 +67,77 @@ class Completions(
     (values, result)
   end completions
 
+  private def toCompletionValues(
+      completion: Completion
+  ): List[CompletionValue.Compiler] =
+    val shouldAddSnippet = path match
+      // if the parent is an apply then we already have ()
```

Comment:
Can you elaborate on it? I couldn't catch it.

Reply:
So if the path to the current position contains an apply it means it's something like `metho@@()` instead of just `metho@@`. I will fix the explanation

Reply:
Added some more comments